nan.h

```diff
@@ -1679,6 +1679,18 @@ inline MaybeLocal<v8::Value> Call(
   return Call(*callback, recv, argc, argv);
 }
 
+inline MaybeLocal<v8::Value> Call(
+    const Nan::Callback& callback
+  , int argc
+  , v8::Local<v8::Value> argv[]) {
```

Comment:
I think this needs handle scopes to avoid leaking the context handle and the global handle.